schema.sql

```diff
@@ -27,6 +28,10 @@ CREATE INDEX IF NOT EXISTS idx_pgpq_tasks_created_at ON pgpq_tasks (created_at A
 
 CREATE INDEX IF NOT EXISTS idx_pgpq_tasks_priority_order ON pgpq_tasks (priority DESC, updated_at ASC);
 
+ALTER TABLE pgpq_tasks ADD COLUMN IF NOT EXISTS not_before TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW();
```

Comment:
Instead of now, can we default to unix(0) time? Same with inserting new rows, pass zero time unless !IsZero()

Reply:
I don't like this idea too much, but OK, done.